traefik/tests/pod-config_test.yaml

```diff
@@ -523,3 +523,37 @@ tests:
             hostPort: 1234
             name: websecure-http3
             protocol: UDP
+  - it: should have a custom list of dns nameservers
+    set:
+      deployment:
+        dnsConfig:
+          nameservers:
+            - 1.1.1.1
+    asserts:
+      - equal:
+          path: spec.template.spec.dnsConfig.nameservers[0]
+          value: "1.1.1.1"
+  - it: should have a list of dns search domain
+    set:
+      deployment:
+        dnsConfig:
+          searches:
+            - ns1.svc.cluster-domain.example
+    asserts:
+      - equal:
+          path: spec.template.spec.dnsConfig.searches[0]
+          value: "ns1.svc.cluster-domain.example"
+  - it: should have a custom dns options
+    set:
+      deployment:
+        dnsConfig:
+          options:
```

Comment:
Same here.

Reply:
done in commit 
https://github.com/traefik/traefik-helm-chart/compare/4a71ed1c3e96e9d44dba31eaf5d7ea2dfd368597..ab50a71f1c86d8bf60c28d1cbf0955166c394cc1